packages/dropdowns/package.json

```diff
@@ -23,6 +23,8 @@
     "@zendeskgarden/container-selection": "^0.2.3",
     "classnames": "^2.2.5",
     "downshift": "^3.2.7",
+    "moment": "^2.24.0",
+    "polished": "^3.4.1",
```

Comment:
This is by the way not small either https://bundlephobia.com/result?p=polished@3.4.1

Reply:
I've updated my import structure to have less of a bundle impact 

```js
import rgba from 'polished/lib/color/rgba';
```

This is already a common dependency with `react-loaders` so it shouldn't be a bundle hit.

Reply:
`polished` is going to be ubiquitous throughout `react-components` as we continue forward with our CSS-in-JS efforts. Not sure if it will make sense as a peer dependency?